dashboard/test/mailers/previews/pd_workshop_mailer_preview.rb

```diff
@@ -54,6 +54,10 @@ def teacher_enrollment_receipt__admin
     mail :teacher_enrollment_receipt, Pd::Workshop::COURSE_ADMIN_COUNSELOR, Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_SLP_CALL1
   end
 
+  def teacher_enrollment_receipt__counselor
```

Comment:
you deleted the old methods. But wanted to mentioned this

If we had a method like
```
def get_attribute_from_slp_call (attribute)
  mail :teacher_enrollment_receipt, attribute, Pd::Workshop::SUBJECT_ADMIN_COUNSELOR_SLP_CALL1
end
```

Reply:
This would have been a bit cleaner
since you could have done this

```
  def teacher_enrollment_receipt__admin
    get_attribute_from_slp_call(Pd::Workshop::COURSE_ADMIN_COUNSELOR,)
  end
```

and so on...

Reply:
Yes good call –– lots of room for improvements here I believe. I think when we're doing more mailer work here in the future, we can look for good ways to improve this